src/main/java/org/sunbird/common/service/ContentServiceImpl.java

```diff
@@ -173,10 +170,34 @@ public SunbirdApiResp getQuestionListDetails(List<String> questionIdList) {
 
 		return null;
 	}
-	
-	public SunbirdApiResp searchLiveContent(String contentId) {
-		SunbirdApiResp response = null;
-		// Call Content Search API here. return the batch details
+
+	public Map<String, Object> searchLiveContent(String contentId) {
+		Map<String, Object> response = null;
+		HashMap<String, String> headerValues = new HashMap<>();
+		headerValues.put(Constants.CONTENT_TYPE, Constants.APPLICATION_JSON);
+		Map<String, Object> filters = new HashMap<>();
+		List<String> primaryCategory = new ArrayList<>(Arrays.asList(Constants.COURSE, Constants.PROGRAM));
+		List<String> status = new ArrayList<>(Arrays.asList(Constants.LIVE));
+		filters.put(Constants.PRIMARY_CATEGORY, primaryCategory);
+		filters.put(Constants.STATUS, status);
```

Comment:
avoid unnecessary local variables... the code should be as follows... 

filters.put(Constants.PRIMARY_CATEGORY, Arrays.asList(Constants.COURSE, Constants.PROGRAM));
filters.put(Constants.STATUS, Arrays.asList(Constants.LIVE));